python/ray/data/_internal/tensor_extensions/pandas.py

```diff
@@ -1018,25 +1026,21 @@ def take(
         extension array to object dtype. This uses the helper method
         :func:`pandas.api.extensions.take`.
 
-        .. code-block:: python
-
-           def take(self, indices, allow_fill=False, fill_value=None):
-               from pandas.core.algorithms import take
-
-               # If the ExtensionArray is backed by an ndarray, then
-               # just pass that here instead of coercing to object.
-               data = self.astype(object)
-
-               if allow_fill and fill_value is None:
-                   fill_value = self.dtype.na_value
-
-               # fill value should always be translated from the scalar
-               # type for the array, to the physical storage type for
-               # the data, before passing to take.
-
-               result = take(data, indices, fill_value=fill_value,
-                             allow_fill=allow_fill)
-               return self._from_sequence(result, dtype=self.dtype)
+        >>> def take(self, indices, allow_fill=False, fill_value=None):  # doctest: +SKIP
+        ...     from pandas.core.algorithms import take
+        ...     # If the ExtensionArray is backed by a ndarray, then
+        ...     # just pass that here instead of coercing to object.
+        ...     data = self.astype(object)
+        ...
+        ...     if allow_fill and fill_value is None:
+        ...          fill_value = self.dtype.na_value
+        ...
+        ...     # fill value should always be translated from the scalar
+        ...     # type for the array, to the physical storage type for
+        ...     # the data, before passing to take.
+        ...     result = take(data, indices, fill_value=fill_value,
+        ...                   allow_fill=allow_fill)
+        ...     return self._from_sequence(result, dtype=self.dtype)  # doctest: +SKIP
```

Comment:
Since this is a long code snippet, I think this is easier to read as a `.. testcode` or `.. code-block`. Could you revert? This is also unrelated to the PR, and might complicate the git blame